articles/azure/azs-how-create-S2S-VPN-azurestack-to-azure.md

```diff
@@ -226,27 +234,31 @@ You'll also need to create a virtual network in Public Azure. This virtual netwo
-
    - **Subscription** - This is your Microsoft Azure subscription.
 
    - **Resource Group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
 
-   - **Location** - Select the location of the Azure resource.
+   - **Name** - The name of the virtual network.
+
+   - **Region** - Select the location of the Azure resource.
+
+       ![Create new virtual network](images/azs-public-browser-new-virtual-network-blade.png)
+
+   - **Address Space** - The virtual network's address range in CIDR notation. This must not overlap the address space of the Azure Stack Hub virtual network.
 
    - **Subnet Name** - The name of the first subnet within the virtual network.
+   
+   - **Subnet Address Range** - The subnet's address range in CIDR notation (for example, 192.168.1.0). It must be within the address range of the virtual network. It should be noted that the address range of a subnet currently in use cannot be edited.
 
-   - **Address Range** - The subnet's address range in CIDR notation (for example, 192.168.1.0). It must be within the address range of the virtual network. It should be noted that the address range of a subnet currently in use cannot be edited.
+        ![Create new virtual network](images/azs-public-browser-new-virtual-network-blade2.png)
 
-   - **DDoS protection** - Select the DDoS protection you would like. For more information, see the [Azure DDoS Protection Standard overview](https://docs.microsoft.com/en-gb/azure/virtual-network/ddos-protection-overview).
+   - **BastionHost** - Select this option to enable BastionHost, which is a new fully platform-managed PaaS service that you provision inside your virtual network. It provides secure and seamless RDP/SSH connectivity to your virtual machines directly in the Azure portal over SSL. When you connect via Azure Bastion, your virtual machines do not need a public IP address.
 
-   - **Service endpoints** - This allows you to enable one or more service endpoints for more information please see [Virtual Network Service Endpoints](https://docs.microsoft.com/en-gb/azure/virtual-network/virtual-network-service-endpoints-overview).
+   - **DDoS protection** - Select this option to enable DDoS protection, which is a paid service that offers enhanced DDoS mitigation capabilities via adaptive tuning, attack notification, and telemetry to protect against the impacts of a DDoS attack for all protected resources within this virtual network. Basic DDoS protection is integrated into the Azure platform by default and at no additional cost.
 
    - **Firewall** - Select this option to enable Azure Firewall, which is a managed cloud-based network security service that protects your Azure Virtual Network resources.
 
-     ![Create new virtual network](images/azs-public-browser-new-virtual-network-blade.png)
+     ![Create new virtual network](images/azs-public-browser-new-virtual-network-blade3.png)
```

Comment:
Image isn't showing. Maybe the case of png?

Reply:
Resolved by fixing file extension.